docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,38 +21,94 @@ name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
-addresses](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xml)
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into three types:
+
+* [Special purpose IPv4 addresses]
+* [Special purpose IPv6 addresses]
+
+### Scope
+
+IP addresses from authoritative name servers are matched against IANAs list of public routable address ranges.
 
 ### Inputs
 
-The domain name to be tested.
+* The domain name to be tested.
+
+### Summary
 
-### Ordered description of steps to be taken to execute the test case
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     | ns_list   | IP addresses in "{ns_list}" listed as globally reachable.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL | ns_list   | None of the IP addresses in "{ns_list}" listed as globally reachable.
```

Comment:
If "ns_list" it can be a really long list and then it will be hard to read the text broken with the name servers in the middle. Suggestion:
```
IP addresses in for the name servers listed are globally reachable. Name servers: {ns_list}
```
Does the second message really need any list of name servers? All name servers will be listed with another tag too.